docs/data/material/customization/density/DensityTool.js

```diff
@@ -71,7 +82,11 @@ export default function DensityTool() {
           </Typography>
         </Grid>
         <Grid item>
-          <IconButton aria-label={t('increaseSpacing')} onClick={decreaseSpacing}>
+          <IconButton
+            aria-label={t('increaseSpacing')}
```

Comment:
Not from your changes, but noticed that it should be decrease spacing.
```suggestion
            aria-label={t('decreaseSpacing')}
```